Makefile

```diff
@@ -25,3 +26,11 @@ build:
 install:
 	cp _output/binaries/colima-$(OS)-$(ARCH) /usr/local/bin/colima
 	chmod +x /usr/local/bin/colima
+
+.PHONY: lint
+lint: ## Run golangci-lint
```

Comment:
I would prefer to assume the environment has golangci-lint installed, rather than have the installer in the Makefile.

Reply:
I can make that change... however the issue with that is each contributor will likely have a different version of golangci-lint or will over time... In my experience it has lead to PRs to fix linting which were newly added.  It also seems valuable to be able to lint in CI and have a standard for identifying which version is the standard.  Do you have a recommendation on how to manage that?  or willing to deal with the consequences?   and how do you expect the CI to define the linter version?

Reply:
ah... actually I think there was an addition to the linter config to specify the version to be expected... please hold :)

Reply:
didn't find it... the consequence remains.   I will remove based on your comment but IMO it will result in checks working on different machines differently and leaves the concern of how to manage the CI linter version

Reply:
it also forces all projects for a developer to be at the same version of linting which is not desirable or expected